sql/catalyst/src/main/antlr4/org/apache/spark/sql/catalyst/parser/SqlBase.g4

```diff
@@ -97,6 +97,8 @@ statement
         SET (DBPROPERTIES | PROPERTIES) tablePropertyList              #setNamespaceProperties
     | ALTER (database | NAMESPACE) multipartIdentifier
         SET locationSpec                                               #setNamespaceLocation
+    | ALTER (database | NAMESPACE) multipartIdentifier
+        SET OWNER ownerType=(USER | ROLE | GROUP) IDENTIFIER           #setNamespaceOwner
```

Comment:
is there any requirement about the owner name? can it contain dot? can it be a string literal like `'abc'`?

Reply:
NO, ONLY `IDENTIFIER` fits. unquoted string with no special chars expect '_'

Reply:
how about non-reserved keywords? These can't be matched by `IDENTIFIER`, we may need to use `identifier`

Reply:
identifier can match more than the hive ones, we may only need  IDENTIFIER + non-reserved

Reply:
identifier matches, thanks for suggestion.